Platform/ElkhartlakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c

```diff
@@ -760,8 +765,10 @@ PlatformFeaturesInit (
     // Update platform specific feature from configuration data.
     PlatformFeatures           = &((PLATFORM_DATA *)GetPlatformDataPtr ())->PlatformFeatures;
     if (PlatformFeatures != NULL) {
-      if (FeaturePcdGet (PcdPreOsCheckerEnabled) && PchIsSciSupported ()) {
+      if (FeaturePcdGet (PcdPreOsCheckerEnabled) && PchIsSciSupported () && !FixedPcdGetBool (PcdIsReleaseBuild)) {

```

Comment:
I have a question about the original code.

Should PlatformFeatures->DebugConsent be 7(Manual) even if FeaturesCfgData->Features.DebugConsent is 0(Disable)?

Reply:
Yes for Fusa sku debug build debug consent is set to 0x7

Reply:
My understanding is
- DebugConsent is 0 by default
- If DebugConsent is enabled, HDA will be disabled to use jtag
- if DebugConsent is enabled and SKU is FUSA, force to set 0x7

Please correct me if my understanding is wrong.

Reply:
yes that's correct
for "DebugConsent is enabled and SKU is FUSA, force to set 0x7" is only valid with debug build

Reply:
I think the implementation should like this.
```
PlatformFeatures->DebugConsent = FeaturesCfgData->Features.DebugConsent;
if ((PlatformFeatures->DebugConsent != 0) && FeaturePcdGet (PcdPreOsCheckerEnabled) && PchIsSciSupported ()) {
  PlatformFeatures->DebugConsent = 7;
}
```

In addition, we have to be able to enable debug consent on release build. In this case, audio will be disabled.